src/mokernels/mokernel.jl

```diff
@@ -4,3 +4,16 @@
 Abstract type for kernels with multiple outpus.
 """
 abstract type MOKernel <: Kernel end
+
+"""
+    matrixkernel(k::MOK, x, y)
+    matrixkernel(k::IndependentMOKernel, x, y(; outputsize))
+
+Convenience function to compute the matrix kernel for two inputs `x` and `y`. The `outputsize` keyword is only required for the `IndependentMOKernel` to indicated the number of outputs. 
```

Comment:
Style: line length

Additionally, it would be helpful to add a doctest here.

Reply:
I have unfortunately never used documenter, will need to read up on this. Style should be fixed by formatter now.

Reply:
```suggestion
Convenience function to compute the matrix kernel for two inputs `x` and `y`. The `outputsize` keyword is only required for the `IndependentMOKernel` to indicate the number of outputs. 
```
Also, how about instead just having
```julia
matrixkernel(k::IndependentMOKernel, x, y) = error("You must provide the number of outputs, call `matrixkernel(k, x, y, out_dim)`")
matrixkernel(k::IndependentMOKernel, x, y, out_dim) = ...
```